.github/workflows/test-language-html2parquet.yml

```diff
@@ -1,7 +1,11 @@
 #
 # DO NOT EDIT THIS FILE: it is generated from test-transform.template,  Edit there and run make to change these files
 #
+<<<<<<<< HEAD:.github/workflows/test-language-html2parquet.yml
```

Comment:
How did you check this in?  I thought git would not allow it.  Anyway.  I recommend the following.
1. merge the latest dev branch into your branch
2. run `make` in this directory and you should get a proper version of this file.